packages/slider/src/presenters/Input.stylesheet.js

```diff
@@ -0,0 +1,172 @@
+const thumbPseudoElements = {
```

Comment:
wondering if we might be able to combine this with `Slider.stylesheet.js` as most components have the one `stylesheet` function.